rest/help.md

```diff
@@ -21,7 +21,7 @@ All of the actions return the same set of data:
 
 # Requirements
 
-varies depending on the API the plugin is interacting with
+* varies depending on the API the plugin is interacting with
```

Comment:
Can you capitalize the first word? Perhaps a requirement is a RESTFUL HTTP/HTTPS resource